queries/ecma/injections.scm

```diff
@@ -119,3 +119,69 @@
   right: 
     (string (string_fragment) @injection.content)
     (#set! injection.language "html"))
+
+;---- Angular injections -----
+; @Component({
+;   template: `<html>`
+; })
+((decorator
+  (call_expression
+    function: ((identifier) @_name
+               (#eq? @_name "Component"))
+    arguments: (arguments
+                 (object
+                   (pair
+                     key: ((property_identifier) @_prop
+                           (#eq? @_prop "template"))
+                     value: ((template_string) @injection.content
+                              (#offset! @injection.content 0 1 0 -1)
+                              (#set! injection.language "angular")
+                            ) 
+                   )
+               ))
+  )
+))
```

Comment:
Please try to follow the (loose) standard format for queries in this repo (no manual alignments, no trailing parens).